open_schools_platform/student_management/students/views.py

```diff
@@ -26,14 +26,15 @@ class StudentProfileApi(ApiAuthMixin, APIView):
     @swagger_auto_schema(
         operation_description="Creates Student profile via provided age, name and family id \n"
                               "Returns Student profile data",
-        request_body=StudentProfileCreateSerializer,
+        request_body=StudentProfileFullSerializer.with_fields(["age", "name", "family", "phone"]),
         responses={201: convert_dict_to_serializer({"student_profile": StudentProfileSerializer()}),
                    404: "No such family",
                    403: "Current user do not have permission to perform this action"},
         tags=[SwaggerTags.STUDENT_MANAGEMENT_STUDENTS]
     )
     def post(self, request):
-        student_profile_serializer = StudentProfileCreateSerializer(data=request.data)
+        student_profile_serializer = StudentProfileFullSerializer.with_fields(["age", "name", "family", "phone"])(
```

Comment:
you can define serializer_class like `self.view_serializer = StudentProfileFullSerializer.with_fields(["age", "name", "family", "phone"])` and reuse it in other places